### PR TITLE
playerctl: Housekeeping

### PR DESCRIPTION
**Pull request by uni-dos**
**Summary**
- add homepage

**Test Plan**

Check homepage tag is in pspec.xml

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

File: packages/p/playerctl/package.yml [hunk 1]
```diff
@@ -18,3 +19,4 @@ build      : |
 install    : |
     %ninja_install
     rm -f $installdir/%libdir%/*.a
+
```
Comment (EbonJaeger) [inline packages/p/playerctl/package.yml:22]: Is this an extra newline?

Comment (uni-dos) [reply packages/p/playerctl/package.yml:22]: Yes, lite-xl inserts a new line when I save. Should I remove it?

Comment (EbonJaeger) [reply packages/p/playerctl/package.yml:22]: The file already ends with a newline, from what I can see. No need to have two newlines.

Comment (uni-dos) [reply packages/p/playerctl/package.yml:22]: On my end it looks like it never had one. It just ended at the rm command

Comment (EbonJaeger) [reply packages/p/playerctl/package.yml:22]: I double-checked in Zed, and yeah, there was already a newline at the end, so this PR adds an extra unneeded line.

**Review by EbonJaeger — CHANGES_REQUESTED** [review]

File: packages/p/playerctl/package.yml (hunk 1 above)
Comment (uni-dos) [reply packages/p/playerctl/package.yml]: For some reason github was hiding that line. Sorry about that.

**Review by ReillyBrogan — APPROVED** [review]

**Pull request merged**